desc/compute/_core.py

```diff
@@ -838,18 +848,43 @@ def compute_geometry(
             )
         )
     if check_derivs("V_r(r)", R_transform, Z_transform):
+        P = compress(  # perimeter
```

Comment:
would this (and major/minor radii) be useful things to add to the data index as well?

Reply:
I don't think so, because it's the perimeter of each toroidal cross-section. I can't think of anything else that would be useful for. And most of the other geometric quantities computed here are 0-dimension scalar values for the whole device.